lib/datadog/tracing/contrib/opensearch/configuration/settings.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require_relative '../../configuration/settings'
+require_relative '../ext'
+
+module Datadog
+  module Tracing
+    module Contrib
+      module OpenSearch
+        module Configuration
+          # Custom settings for the OpenSearch integration
+          # @public_api
+          class Settings < Contrib::Configuration::Settings
+            option :enabled do |o|
+              o.default { env_to_bool(Ext::ENV_ENABLED, true) }
+              o.lazy
+            end
+
+            option :analytics_enabled do |o|
+              o.default { env_to_bool(Ext::ENV_ANALYTICS_ENABLED, false) }
+              o.lazy
+            end
+
+            option :analytics_sample_rate do |o|
+              o.default { env_to_float(Ext::ENV_ANALYTICS_SAMPLE_RATE, 1.0) }
+              o.lazy
+            end
+
+            option :quantize, default: {}
+
+            option :service_name do |o|
+              o.default do
+                Contrib::SpanAttributeSchema.fetch_service_name(
```

Comment:
Curious to see this in here. My understanding is this (`SpanAttributeSchema`) was still on a feature branch? Or is this really in `master`? cc @zarirhamza

Reply:
Answering my question, this looks like it was a prototype of sorts merged months ago.